src/uwtools/tests/test_cli.py

```diff
@@ -110,7 +110,7 @@ def test__check_file_vs_format_pass_explicit():
     assert args[STR.infmt] == fmt
 
 
-@pytest.mark.parametrize("fmt", vars(FORMAT).keys())
+@pytest.mark.parametrize("fmt", FORMAT.formats())
```

Comment:
I added a static method on the `FORMAT` dataclass to return a map from format variant names (e.g. `yaml` and `yml`) to their canonical names (e.g. `yaml`) and use it here to avoid getting unwanted attribute names from `vars(FORMAT)`.